drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,108 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api");
+for (var control of data.controls) {
+    // do something with the _control_
+}
+```
+
+### Details
+We can imaging a situation, when an application has a base URL provided, i.e. in a configuration or in the input field.
+Once provided, I'd expect the application to connect to the entry-point of the Web API and download initial
```

Comment:
Hmm - I acted from an product owner somehow :). Anyway, I'll make it impersonal as you suggested.